utils/utils.go

```diff
@@ -22,6 +23,22 @@ func ExtractTextFromURL(url string) (string, error) {
 	return readableText, nil
 }
 
+func IfFileWithAPiKeyExists(apiKeyPath string) bool {
+	exists, err := fs.PathExists(apiKeyPath)
+	if err != nil {
+		return false
```

Comment:
you should be returning bool, error. Error does not mean false in this context, is mean error :D